src/bin/cargo/commands/add.rs

```diff
@@ -72,6 +72,10 @@ The package will be removed from your features.")
 Example uses:
 - Depending on multiple versions of a crate
 - Depend on crates with the same name from different registries"),
+            flag(
+                "ignore-rust-version",
+                "Ignore `rust-version` specification in packages (unstable)"
```

Comment:
Looking around, it seems like we document our unstable flags in the command reference.  We should probably reflect this flag in there.